crates/trigger/src/runtime_config.rs

```diff
@@ -259,6 +261,52 @@ fn resolve_config_path(path: &Path, config_opts: &RuntimeConfigOpts) -> Result<P
     Ok(base_path.join(path))
 }
 
+pub(crate) struct SummariseRuntimeConfigHook {
+    runtime_config_file: Option<PathBuf>,
+}
+
+impl SummariseRuntimeConfigHook {
+    pub(crate) fn new(runtime_config_file: &Option<PathBuf>) -> Self {
+        Self {
+            runtime_config_file: runtime_config_file.clone(),
+        }
+    }
+}
+
+impl TriggerHooks for SummariseRuntimeConfigHook {
+    fn app_loaded(
+        &mut self,
+        _app: &spin_app::App,
+        runtime_config: &RuntimeConfig,
+        _resolver: &Arc<spin_expressions::PreparedResolver>,
+    ) -> anyhow::Result<()> {
+        if let Some(path) = &self.runtime_config_file {
+            let mut opts = vec![];
+            for opt in runtime_config.opts_layers() {
+                for id in opt.key_value_stores.keys() {
+                    opts.push(format!("[key_value_store.{id}]"));
```

Comment:
Should these also include types? That would be helpful for validating the right version of a config between dev/deployments

Reply:
@endocrimes Could you be specific about what you mean by "include types"?  Are you thinking e.g. whether the KV backing store is SQLite vs Redis vs Cosmos, etc.?

Reply:
Exactly!

Reply:
Added backing type info.  It now looks like:

```
$ spin up -f ~/testing/kvmadness/ --runtime-config-file ~/testing/kvmadness/runtime-config.toml 
Logging component stdio to "/home/ivan/testing/kvmadness/.spin/logs/"
Using [key_value_store.default: spin], [key_value_store.bibblybobbly: spin] runtime config from "/home/ivan/testing/kvmadness/runtime-config.toml"
Storing default key-value data to ".spin/my_super_duper_store.db"
Storing default SQLite data to "/home/ivan/testing/kvmadness/.spin/sqlite_db.db"
```

I am not sure this is the perfect layout - it feels a bit spindly - but we can presumably iterate without causing a disaster.  But if folks hate it on sight I'm open to other ideas!